waspc/ChangeLog.md

```diff
@@ -6,6 +6,10 @@
 
 - In the `usernameAndPassword` authentication method, the function `login()` imported from `wasp/client/auth` now accepts an object with `username` and `password` instead of two separate arguments ([#2598](https://github.com/wasp-lang/wasp/pull/2598))
 
+### 🐞 Bug fixes
+
+- Calling `useAuth()` when the user is not logged in will not fail the request, only return `null` ([#2689](https://github.com/wasp-lang/wasp/pull/2689))
```

Comment:
Hmm, is this a breaking change too? Could users have relied on this behavior?

Reply:
Fair enough, no breaking change then.

If someone does rely on it and we break their code, we'll teach them about [Hyrum's law](https://www.hyrumslaw.com/) as an aplogy :)